script/Arcueid/Day 9/09_00_ARC09_2.txt

```diff
@@ -70,6 +70,7 @@ I should save that for tonight and discuss them with Arcueid.
 [sha:872894ed81cd6df96c42e8e937e7ac2c51c35f39]{
 -- Page 30, Offset 21394.
 -- 昼間ぐらいは自分の日常に戻らないと、俺もどうにかしてしまう。
+//Instinctually, I would never use drastic here. In the english absurd sounds good?
```

Comment:
As in, replace drastic with absurd? I think the intent here is "I'll do something I might regret later", so absurd doesn't really fit in that context. Possible "do something drastic" is UK speak, if so you could maybe naturalize it to "it might really push me over the edge" or something.

Reply:
I think the current TL is good for どうにかしてしまう。